### PR TITLE
Update scmsync documentation

### PR DESCRIPTION
**Pull request by adrianschroeter**
changed defaults and changed recommendations

### CODE REVIEW

File: xml/obs_scm_bridge.xml [hunk 1]
```diff
@@ -12,31 +12,35 @@
...
+      instance. However, any SCM server reachable via network could be used.</para>
 
-      <para>osc can be used to checkout sources. This will create a scm repository, so any modification needs to be
+      <para>osc can still be used to checkout sources. This will create a scm repository, so any modification needs to be
       done using the native scm tooling (eg. git). osc can still be used for local building and checking server
       side build results.</para>
 
-      <para>Be aware that any source mechanics like link or branch of OBS are not working when using the bridging.
-      This means also that workflows are not working anymore or are limited. For example a submit request would
+      <para>Be aware that any source modifican mechanics submit requests or link behaviour may not work
```
Comment (danidoni) [inline xml/obs_scm_bridge.xml:35]: `modifican` ?

File: xml/obs_scm_bridge.xml [hunk 2]
```diff
@@ -115,6 +126,16 @@
 	honored by OBS, but it can be part of the git tree anyway for pbuild users.
         </para>
       </sect3>
+
+      <sect3 xml:id="sec.obs.obs_scm_bridge.setup.specials.fork">
+        <title>Forking a scmsync package</title>
```
[the remark this answers is not in the document]
Comment (adrianschroeter) [reply xml/obs_scm_bridge.xml:130]: really? I am no native speaker, but first google says also that a is used before a word with "s"?

[the remark this answers is not in the document]
Comment (danidoni) [reply xml/obs_scm_bridge.xml:130]: It's about the sound more than if it's a vowel or a consonant (see [here](https://www.merriam-webster.com/grammar/is-it-a-or-an))

File: xml/obs_scm_bridge.xml [hunk 3]
```diff
@@ -12,31 +12,34 @@
...
 
   <sect1 xml:id="sec.obs.obs_scm_bridge">
     <title>SCM Bridge</title>
 
     <sect2 xml:id="sec.obs.obs_scm_bridge.setup.introduction">
       <title>Introduction</title>
 
-      <para>The SCM bridge allows to setup single packages or entire projects in any trusted SCM repositories. 
+      <para>The SCM bridge allows to setup single package or entire project sources to be stored and
+            maintained in any trusted SCM repositories.
```
Comment (smithfarm) [inline xml/obs_scm_bridge.xml:24]: It's a good idea, when introducing a new acronym into a text, to provide an expansion of the acronym. 

I'm guessing this is the first instance of "SCM" in the document's text, so we could change "SCM bridge" to "SCM (source control management) bridge" here?

File: xml/obs_scm_bridge.xml [hunk 4]
```diff
@@ -12,31 +12,34 @@
...
+      <para>The SCM bridge allows to setup single package or entire project sources to be stored and
+            maintained in any trusted SCM repositories. 
             However, the only supported SCM is currently git.</para>
 
-      <para>This allows to manage the entire package sources in an external SCM repository. While the classic
-      OBS services for scms (like obs_scm or tar_scm) are using the workflows inside of OBS, these workflows
-      have to happen inside of the SCM provider instance. OBS will just verify the authenticity of that resource in
-      future.</para>
+      <para>This allows to run the entire packaging source update in an external SCM repository using the
+      native tooling. It is recommended to have this SCM server on the same trust level as the OBS
```
Comment (smithfarm) [inline xml/obs_scm_bridge.xml:28]: The first sentence of this paragraph is potentially unclear in two ways.

First, it refers to "the entire packaging source update" - the definite article ("the") indicates that there is some definite (specific) packaging source update that is being referred to, but I'm not sure this is the case? What do you mean by "the entire packaging source update", here, exactly?

Second, it says "using the native tooling". Again, the definite article is used, but I'm not sure the reader knows which "native tooling" you mean here?

File: xml/obs_scm_bridge.xml [hunk 5]
```diff
@@ -12,31 +12,34 @@
...
-      <para>Be aware that any source mechanics like link or branch of OBS are not working when using the bridging.
-      This means also that workflows are not working anymore or are limited. For example a submit request would
-      still work when having scmsync on package level, but the user would manually specify the target always.
-      It won't work at all when the entire project is managed via git. This briding mechanic is designed to follow
-      the configured git provider, so this instance need to provide any required workflow.</para>
+      <para>Be aware that any source modification mechanics, submit requests or link behaviour may work
+      different or not all. For example you can not use automatic package source merging via _link files anymore.
+      This means also that workflows inside of OBS are not working anymore or are limited. A submit or release
+      request would still work with a scmsync source, but the user would manually need to specify the target always.
+      Functionality for project managed resources is even more limited.</para>
```
Comment (smithfarm) [inline xml/obs_scm_bridge.xml:39]: Can you give an example or two of these "project managed resources" for readers who aren't familiar with this term?

File: xml/obs_scm_bridge.xml [hunk 6]
```diff
@@ -127,7 +147,7 @@
       by referencing it via the scmsync meta tag.
       </para>
       <para>
-      This has been only implemented for gitea atm, you can find details in the <link xlink:href="https://github.com/adrianschroeter/obs-gitea-bridge">obs-gitea-bridge</link> documentation.
+      This has been only implemented for gitea atm, you can find details in the <link xlink:href="https://github.com/openSUSE/obs-gitea-bridge">obs-gitea-bridge</link> documentation.
```
Comment (smithfarm) [inline xml/obs_scm_bridge.xml:150]: The SCM bridge has so far only been implemented for gitea. You can find details in . . .

Comment (adrianschroeter) [reply xml/obs_scm_bridge.xml]: not the scmsync, but the update notification. changed that now.

File: xml/obs_scm_bridge.xml (hunk 5 above)
Comment (adrianschroeter) [reply xml/obs_scm_bridge.xml]: I have reworeded this now.

File: xml/obs_scm_bridge.xml [hunk 7]
```diff
@@ -12,31 +12,34 @@
...
       done using the native scm tooling (eg. git). osc can still be used for local building and checking server
       side build results.</para>
 
-      <para>Be aware that any source mechanics like link or branch of OBS are not working when using the bridging.
-      This means also that workflows are not working anymore or are limited. For example a submit request would
-      still work when having scmsync on package level, but the user would manually specify the target always.
-      It won't work at all when the entire project is managed via git. This briding mechanic is designed to follow
-      the configured git provider, so this instance need to provide any required workflow.</para>
+      <para>Be aware that any source modification mechanics, submit requests or link behaviour may work
+      different or not all. For example you can not use automatic package source merging via _link files anymore.
```
[the remark this answers is not in the document]
Comment (adrianschroeter) [reply xml/obs_scm_bridge.xml]: changed that way

File: xml/obs_scm_bridge.xml (hunk 4 above)
Comment (adrianschroeter) [reply xml/obs_scm_bridge.xml]: tried to be more clear now.

**Review by smithfarm — APPROVED** [review]

**Pull request merged**